bluebottle/tasks/taskmail.py

```diff
@@ -212,6 +212,30 @@ def new_reaction_notification(sender, instance, created, **kwargs):
     mailer.send_mail()
 
 
+@receiver(pre_save, weak=False, sender=Task)
+def email_deadline_update(sender, instance, **kwargs):
+    if instance.pk:
+        previous_instance = Task.objects.get(pk=instance.pk)
+
+        if (previous_instance.deadline != instance.deadline and
+                instance.status not in (Task.TaskStatuses.realized, Task.TaskStatuses.closed)):
+            for task_member in instance.members_applied:
+
+                with TenantLanguage(task_member.member.primary_language):
+                    subject = _('The deadline of your task is changed')
+
+                send_mail(
+                    template_name='tasks/mails/deadline_changed.mail',
+                    subject=subject,
+                    title=instance.title,
+                    original_date=previous_instance.deadline,
+                    date=instance.deadline,
+                    to=task_member.member,
+                    site=tenant_url(),
+                    link='/go/tasks/{0}'.format(instance.id)
+                )
+
+
```

Comment:
Hm, I would say that the signal should go in `signals.py`, but since we already did like this it's fine for now...